tests/unit-tests/python/foglamp_test/services/core/api/test_scheduler.py

```diff
@@ -52,15 +52,15 @@ class TestScheduler:
     def setup_class(cls):
         asyncio.get_event_loop().run_until_complete(add_master_data())
         from subprocess import call
-        call(["foglamp", "start"])
+        call(["scripts/foglamp", "start"])
         # TODO: Due to lengthy start up, now tests need a better way to start foglamp or poll some
         #       external process to check if foglamp has started.
         time.sleep(30)
 
     @classmethod
     def teardown_class(cls):
         from subprocess import call
-        call(["foglamp", "stop"])
+        call(["scripts/foglamp", "stop"])
```

Comment:
there is nothing called foglamp stop. We need to mock the foglamp start/stop commands to make it a pure unit test

Reply:
let's comment it, and add a TODO for now!